### PR TITLE
Mongo v5

### PR DESCRIPTION
**Pull request by gustawdaniel**
goal of this PR is to solve

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: mongo-migrate@0.0.1
npm ERR! Found: mongodb@5.7.0
npm ERR! node_modules/mongodb
npm ERR!   mongodb@"^5.7.0" from mongo-migrate@0.0.1
npm ERR!   apps/mongo-migrate
npm ERR!     mongo-migrate@0.0.1
npm ERR!     node_modules/mongo-migrate
npm ERR!       workspace apps/mongo-migrate from the root project
npm ERR!   peer mongodb@">=3.4.0" from mongodb-client-encryption@2.9.0
npm ERR!   node_modules/mongodb-client-encryption
npm ERR!     peerOptional mongodb-client-encryption@">=2.3.0 <3" from mongodb@5.7.0
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer mongodb@"4.x.x" from mongo-migrate-ts@1.1.4
npm ERR! node_modules/mongo-migrate-ts
npm ERR!   mongo-migrate-ts@"^1.1.4" from mongo-migrate@0.0.1
npm ERR!   apps/mongo-migrate
npm ERR!     mongo-migrate@0.0.1
npm ERR!     node_modules/mongo-migrate
npm ERR!       workspace apps/mongo-migrate from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^9.0.0",
     "@semantic-release/release-notes-generator": "^10.0.3",
-    "@types/cli-table": "^0.3.1",
+    "@types/cli-table": "^0.3.0",
```
Comment (gustawdaniel) [inline package.json]:
```suggestion
    "@types/cli-table": "^0.3.1",
```

**Comment by gustawdaniel** [comment]
fixed 

yarn audit v1.22.19
53 vulnerabilities found - Packages audited: 989

merge should close https://github.com/mycodeself/mongo-migrate-ts/issues/86

**Review by mycodeself — APPROVED** [review]
Hey @gustawdaniel

thanks a lot for your contribution, I'm going to merge this and new release will be available.

**Pull request merged**